MergerService/src/run.cs

```diff
@@ -358,6 +360,16 @@ private void RunTask(MergeTask task, ITaskUtils taskUtils, string? managerCallba
                                     if (blob != null)
                                     {
                                         tiles.Add(new Tile(coord, blob));
+                                        currentTilesChunkBytes += blob.Length;
+
+                                        if (currentTilesChunkBytes >= this._targetTilesChunkBytesMax || tiles.Count >= this._batchSize)
```

Comment:
I think you might still want to have some kind of limit on the amount of tiles handled each time (probably should be much larger then 1000, if it is limited by the total amount of bytes as well)
just to avoid different unexpected issues that might happen when trying to write too many tiles/files at once
but its probably a good idea to keep this option optional (maybe if 0 or -1 is passed then it is ignored?)
let me know what you prefer (or if you want to discuss it in person)

Reply:
You make a good point. But once we make a task resume-able (which is planned in the future) then I feel that the amount of tiles in a batch doesn't really matter. Unless there will be a type of `Data` that the amount of objects inserted in one go will affect the upload time drastically.
To sum this up, it may be a good option to make it optional as you suggested. I fear that setting it to 0 or -1 will not be explicit enough, maybe we should add `limitBatchSize` boolean env to indicate the use of `batchSize` as part of the limit when always considering `memory`.

Reply:
done, added `limitBatchSize`